src/yang/ietf-bgp-rib.yang

```diff
@@ -142,6 +142,16 @@ submodule ietf-bgp-rib {
         reference
           "RFC 4271: Section 5.1.3.";
       }
+      leaf link-local-next-hop {
```

Comment:
This is under the common grouping of attr-set-attributes. How do we control it for safis where this is not relevant? we need a conditional check?

Reply:
Our choices are:
1. It's an optional leaf, so it simply drops out when not in use.
2. We can add a conditional clause that limits it for ipv6 address families.  However, that starts to entangle maintenance of this leaf based on address families that may not yet be at this point.  Recall that our definition is an identity that is the tuple.

I'm up for better ideas.

Reply:
Why is an optional leaf not a good choice?

Reply:
In further review, I believe these leaves are only available in a read-only context.  If this was a read-write context for config state, it might be more problematic.

Reply:
Ok with any of the option suggested. Thinking a bit more, we can also move this as a backend validation check (along with or without option 1 or 2).

Reply:
I think we've converged on this point.  The answer is we just leave it and it only is used when required. 
If so, please approve review.